disco_aws_automation/disco_dynamodb.py

```diff
@@ -132,3 +141,153 @@ def _convert_table_to_dict(table):
                 "StreamSpecification": table.stream_specification,
                 "LatestStreamLabel": table.latest_stream_label,
                 "LatestStreamArn": table.latest_stream_arn}
+
+
+class AsiaqDynamoDbBackupManager(object):
+    """
+    Management tool for creating, restoring from, and inspecting DynamoDB table backups.
+    """
+    S3_BUCKET_KEY = "dynamodb-backup"
+    BACKUP_DESC_TEMPLATE = "Periodic backup for %s table in %s env"
+
+    BACKUP_PIPELINE_TEMPLATE = 'dynamodb_backup'
+    RESTORE_PIPELINE_TEMPLATE = 'dynamodb_restore'
```

Comment:
What's the use case?  This class is specifically to manage creating and activating pipelines for backups—if people want to create arbitrary pipelines, the data pipeline tooling supports that, but I don't see an obvious benefit to adding the template names to the list of things that users need to know in order to create a backup.